helm/sealed-secrets/values.yaml

```
@@ -12,6 +12,10 @@ fullnameOverride: ""
 ## @param namespace Namespace where to deploy the Sealed Secrets controller
 ##
 namespace: ""
+## @param revisionHistoryLimit Number of old history to retain to allow rollback (If not set, default Kubernetes value is set to 10)
+## e.g:
+## revisionHistoryLimit: 1
```

```suggestion
revisionHistoryLimit: ""
```

I would also move this a bit down, under the "Sealed Secrets Parameters" section.

Could you please add this parameter to the readme too?

---

Thank you for your feedback @alemorcuq. Code has been adapted.